packages/config/src/index.ts

```diff
@@ -402,6 +402,18 @@ export default async (
   } else {
     pnpmConfig.extraBinPaths = []
   }
+
+  if (pnpmConfig.preferSymlinkedExecutables && pnpmConfig.virtualStoreDir) {
+    pnpmConfig.extraEnv = {
+      NODE_PATH: path.join(
+        pnpmConfig.lockfileDir ?? pnpmConfig.dir,
+        'node_modules',
+        pnpmConfig.virtualStoreDir,
```

Comment:
I don't think this is how virtualStoreDir location is calculated. Maybe check in the get-context package.